components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -128,27 +146,58 @@ export default function () {
                             })}
                             {activeWorkspaces.length > 0 && <div className="py-6"></div>}
                             {inactiveWorkspaces.length > 0 && (
-                                <div className="p-3 text-gray-400 bg-gray-50 dark:bg-gray-800 rounded-xl text-sm text-center">
-                                    Unpinned workspaces that have been inactive for more than 14 days will be
-                                    automatically deleted.{" "}
-                                    <a
-                                        className="gp-link"
-                                        href="https://www.gitpod.io/docs/life-of-workspace/#garbage-collection"
+                                <div className="pt-14">
+                                    <div className="border-t border-gray-200 dark:border-gray-800"></div>
+                                    <div
+                                        onClick={() => setShowInactive(!showInactive)}
+                                        className="flex cursor-pointer py-6"
                                     >
-                                        Learn more
-                                    </a>
+                                        <div className="flex flex-col">
+                                            <h2 className="">Inactive Workspaces</h2>
```

Comment:
**issue(non-blocking)**: While the heading here describes the logical separation from the active workspaces, there's no equivalent heading for the active workspaces, thus making this slightly confusing. Instead, using a clear verb-based action here could help.

```suggestion
                                            <h2 className="">Show Inactive Workspaces</h2>
```

Reply:
Minor after thought here. If we're going with the more explicit verb action here, we would need to also change this to _Hide Inactive Workspaces_ when the collapsible element is expanded.

Reply:
Marking as resolved. Consensus was to drop this idea. 🤝